code/datums/ai_law_sets.dm

```diff
@@ -1,4 +1,6 @@
-/******************** Asimov ********************/
+// DO NOT TOUCH ANY LAWS WITHOUT SYNTHLORE APPROVAL
```

Comment:
I have no interest in comments like that being spammed all over the code base.

In addition this is definitely not under the exclusive purview of synt lore.